homeassistant/components/vacuum/xiaomi_miio.py

```diff
@@ -21,7 +21,7 @@
     ATTR_ENTITY_ID, CONF_HOST, CONF_NAME, CONF_TOKEN, STATE_OFF, STATE_ON)
 import homeassistant.helpers.config_validation as cv
 
-REQUIREMENTS = ['python-miio==0.3.2']
+REQUIREMENTS = ['python-miio>=0.3.2']
```

Comment:
We are pinning the releases. The reason to do this is that we know that the platform was working with the given release and to avoid side effects.

If a new release is available then the requirement should be updated and the platform be tested.